test-cases/PR-provision-test-docker.yaml

```diff
@@ -7,6 +7,8 @@ n_loaders: 1
 n_monitor_nodes: 1
 n_db_nodes: 1
 
+instance_type_runner: c5.2xlarge
```

Comment:
why not something much smaller like c5.large?

Reply:
2 cpus isn't enough to running the docker backend (also the integration test) reliably.

Reply:
those are optional CI phase that take ~30min each

Reply:
oh it's running locally on this machine as docker backend.
I agree.